### PR TITLE
Object navigation (prev and next links) in modules view

### PR DESCRIPTION
**Pull request by didoda**
Ref: #326

### CODE REVIEW

File: src/Controller/AppController.php [hunk 1]
```diff
@@ -277,4 +278,51 @@ protected function applySessionFilter() : ?Response
...
+            $objectNav[$moduleName][$object['id']] = [
+                'prev' => ($i > 0) ? Hash::get($objects, sprintf('%d.id', $i - 1)) : null,
+                'next' => ($i + 1 < $total) ? Hash::get($objects, sprintf('%d.id', $i + 1)) : null,
+                'index' => $i + 1,
+                'total' => $total,
+                'object_type' => Hash::get($objects, sprintf('%d.object_type', $i)),
+            ];
+        }
+        $session = $this->request->getSession();
+        $session->write('objectNav', $objectNav);
```
Comment (stefanorosanelli) [inline src/Controller/AppController.php:311]: Using `objectNav.$moduleName` as key? We don't need `objectTypeNav` and we could store more lists at the same time (e.g. navigate items in different tabs..)

File: src/View/Helper/LinkHelper.php [hunk 2]
```diff
@@ -233,4 +234,44 @@ protected function findFiles(array $filter, string $type) : array
...
+        $next = '›';
+        $objectType = Hash::get($data, 'object_type', null);
+        if (!empty($data['prev'])) {
+            $prev = $this->Html->link(
+                $prev,
+                [
+                    '_name' => 'modules:view',
+                    'object_type' => $objectType,
+                    'id' => $data['prev'],
+                    'title' => __('prev'),
```
Comment (stefanorosanelli) [inline src/View/Helper/LinkHelper.php]: Why add `?title=next` to query string?

**Review by stefanorosanelli — CHANGES_REQUESTED** [review]
Overall ok, good job!
Just a couple of minor suggestions/doubts

File: src/Controller/AppController.php (hunk 1 above)
Comment (didoda) [reply src/Controller/AppController.php:311]: We need the info stored in `objectTypeNav`... it tells the view which was the module index recently used.
The `currentModuleName` is not a good choice... think about a navigation in "objects" index, and then to a view of a "document".  In this case `objectTypeNav = 'objects'` and `currentModuleName = 'documents'`. The object navigation should use `'objects'` (and it does, using `objectTypeNav`)

File: src/View/Helper/LinkHelper.php (hunk 2 above)
Comment (didoda) [reply src/View/Helper/LinkHelper.php]: This is something coming from BEdita3 solution... Let's get rid of it

**Review by stefanorosanelli — APPROVED** [review]

**Pull request merged**